app/Models/Role.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Role extends Model
+{
+    use SoftDeletes;
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table="roles";
+    /**
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Models/Role.php</br>String "roles" does not require double quotes; use single quotes instead